frontend/ui/data/css_tweaks.lua

```diff
@@ -134,6 +134,11 @@ h1 + h6, h2 + h6, h3 + h6, h4 + h6, h5 + h6 { page-break-before: avoid !importan
                     ]],
                 },
             },
+            {
+                id = "page-break-after-body";
+                title = _("New page on next <body>"),
```

Comment:
I think this'd sound slightly better.
```suggestion
                title = _("New page before next <body>"),
```

Reply:
I repeated the approved "New page on headings".
New page starts on the beginning of the new body.